crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -37,6 +37,9 @@ def upgrade
         crowbar_files
       end
 
+      if @status[:errors].any?
+        @backup.errors.add(:upgrade, @status[:errors])
+      end
```

Comment:
@rsalevsky not sure if that is necessary here, do you think it makes sense to add the errors to the model? and does it make a difference that the model object is cached here?